### ISSUE TITLE
Consider missing values warning only for numeric comparisons

### ISSUE THREAD

**Issue by kbjarkefur** [worldbank/stata-linter#65]
This is not needed for first version, but I think that ```if "`mylocal'" != ""``` should not have to result in the warning `(Remember that "a != 0" or "a > 0" include cases where a is missing.)`. This warning is accurate for numeric comparison. But I think it risk being confusing if it is generated for string comparisons.
